### PR TITLE
Update License Year

### PR DESCRIPTION
**Pull request by AyushBherwani1998**
## Description

Updated year of License from 2017 to 2018 to keep it up to date.

## Deploy Notes

Notes regarding deployment the contained body of work. These should note any db migrations, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,15 +24,15 @@ To generate code coverage, set the `COVERAGE` environment variable when running
 COVERAGE=1 bundle exec rspec
 ```
 
-This will create a folder named `coverage`. Open `index.html` in this folder to view coverage statistics.
+This will create a folder named `coverage`. Open `index.html` in the coverage folder to view coverage statistics.
```
Comment (tbedford) [inline README.md]: We might as well put `coverage` in backticks again.

File: LICENSE.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Nexmo
+Copyright (c) 2018 Nexmo
```
Comment (tbedford) [inline LICENSE.txt]: Should probably now be a range so Copyright (c) 2017-2018 Nexmo.

**Comment by AyushBherwani1998** [comment]
@tbedford  Done the required changes. Thanks.

**Comment by AyushBherwani1998** [comment]
@tbedford done all the requested changes in.

**Comment by AyushBherwani1998** [comment]
@tbedford Done all the requested changes.

**Pull request merged**